amethyst_ui/src/prefab.rs

```diff
@@ -499,11 +617,17 @@ where
     None
 }
 
-type UiPrefabData<A = AudioFormat, I = TextureFormat, F = FontFormat> = (
+type UiPrefabData<
+    A = AudioFormat,
+    I = TextureFormat,
+    F = FontFormat,
+    CD = <NoCustomUi as ToNativeWidget>::PrefabData,
```

Comment:
Idiomatic rust suggests we use single letter type parameters.

Reply:
Updated